gensim/models/base_any2vec.py

```diff
@@ -300,6 +302,64 @@ def _set_train_params(self, **kwargs):
     def __init__(self, sentences=None, workers=3, vector_size=100, epochs=5, callbacks=(), batch_words=10000,
                  trim_rule=None, sg=0, alpha=0.025, window=5, seed=1, hs=0, negative=5, cbow_mean=1,
                  min_alpha=0.0001, compute_loss=False, fast_version=0, **kwargs):
+        """Construct a base word embeddings model.
+
+        Parameters
+        ----------
+        sentences : iterable of iterable of str
+            Can be simply a list of lists of tokens, but for larger corpora,
+            consider an iterable that streams the sentences directly from disk/network.
+            See :class:`~gensim.models.word2vec.BrownCorpus`, :class:`~gensim.models.word2vec.Text8Corpus`
+            or :class:`~gensim.models.word2vec.LineSentence` in :mod:`~gensim.models.word2vec` module for such examples.
+        workers : int
+            Number of working threads, used for multiprocessing.
+        vector_size : int
+            Dimensionality of the feature vectors.
+        epochs : int
+            Number of iterations (epochs) of training through the corpus.
+        callbacks : list of :class: `~gensim.models.callbacks.CallbackAny2Vec`, optional
+            List of callbacks that need to be executed/run at specific stages during training.
+        batch_words : int
+            Number of words to be processed by a single job.
+        trim_rule : function, optional
```

Comment:
Need to describe function signature (in the description of the parameter). I see ` (word, count, min_count)`, but I need types here + show it more explicitly  (maybe new sentence)